.github/workflows/institution-e2e-tests.yml

```diff
@@ -0,0 +1,52 @@
+name: Institution E2E
+
+on: pull_request
+
+jobs:
+  setup-env:
+    name: 'Load ENV Vars'
+    uses: ./.github/workflows/setup-env.yml
+    secrets: inherit
+
+  e2e-tests:
+    name: Cypress (Institutions)
+    runs-on: ubuntu-latest
+    needs: [setup-env]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: "lts/*"
+          check-latest: true
+      - run: npm ci
+
+      - name: 'Create env file'
+        run: |
+          ENV_FILE_PATH=./apps/institution-service/.env
+          touch ${ENV_FILE_PATH}
+          
+          # Vars
+          echo -e "${{ needs.setup-env.outputs.env_vars }}" >> ${ENV_FILE_PATH}
+          echo PORT="8088" >> ${ENV_FILE_PATH}
```

Comment:
Need to remove this. Probably in the auth service action too

Reply:
Let's talk about how we decide on port numbers. The auth service uses .env for this. We can do static, but we should decide the best way to proceed. Do they need to be different, or does that not matter, etc.